Package.swift

```diff
@@ -18,7 +18,9 @@ let package = Package(
       name: "tibs",
       targets: ["tibs"])
   ],
-  dependencies: [],
+  dependencies: [
+    .package(url: "https://github.com/swiftlang/swift-lmdb.git", branch: "main"),
+  ],
```

Comment:
Could you use a path based dependency if `SWIFTCI_USE_LOCAL_DEPS` is set? 

Similar to https://github.com/swiftlang/sourcekit-lsp/blob/c23b4ff776d34479ddc37d9d2133e0e7007ad35f/Package.swift#L571

Reply:
Done.